src/test/java/com/bio4j/angulillos/TwitterGraphTestSuite.java

```diff
@@ -14,8 +14,9 @@
 
   public TwitterSchema<RV,RE>.User.Vertex addUser(String name, Integer age) {
 
-    return g.addVertex(g.user)
-            .set(g.user.name, name)
+    TwitterSchema<RV,RE>.User.Vertex u = g.user.addVertex();
```

Comment:
In the end I think we still need to write `VertexType<VT, ...>.Vertex` instead of just `VT.Vertex` in the abstract code. because on this line I get 

```
[warn] /Users/laughedelic/dev/bio4j/angulillos/src/test/java/com/bio4j/angulillos/TwitterGraphTestSuite.java:17: unchecked conversion
[warn]   required: com.bio4j.angulillos.VertexType<com.bio4j.angulillos.TwitterSchema<RV,RE>.User,com.bio4j.angulillos.TwitterSchema<RV,RE>,RV,RE>.Vertex
[warn]   found:    com.bio4j.angulillos.VertexType.Vertex
[warn]     TwitterSchema<RV,RE>.User.Vertex u = g.user.addVertex();
```

and it means that the `VT.Vertex addVertex()` signature is interpreted as `VertexType.Vertex addVertex()`, because at that moment everything the compiler know is that `VT <:  VertexType` (without params).